test/support/performance.js

```diff
@@ -32,8 +32,10 @@
             var self = this;
             testSupport.requireBenchmarkBuild(this.icalVersion, function(lib) {
               self.icalObject = lib;
-              if (!self.icalObject) {
+              if (!self.icalObject || Object.keys(self.icalObject).length === 0) {
                 console.log('Version ICAL_' + self.icalVersion + ' not found, skipping');
+                self.icalObject = null;
+                return callback(null);
```

Comment:
Just curious, why is this callback needed?

Reply:
not sure tbh, I just noticed that these 2 lines are used below but are missing from here.

Reply:
Just to avoid duplication, you could throw here (in the if), and the same error will be logged by the catch? Does that make sense?

Reply:
yeah ok 👍